### DISCUSSION TITLE
What does the "/" in Instances, Modules and Macros indicate?

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#4451] [General]
For instance, what does "5/45" mean?

It is a bit hard to read, it would have been easier if there were two columns for the "Instances" numbers...

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/463b3945-3120-4da2-a36a-43152809a47a)

**Comment by maliberty** [comment]
```<number of objects in this module>/<number of objects in this module and all its recursive children>```

The tradeoff of more columns is that it takes more screen real estate.  Its already a pretty wide window.

**Reply by oharboe** [reply]
I see. I'm mostly interested in reading the leftmost number, so it's hard to read.

**Reply by oharboe** [reply]
@maliberty Though... The columns can be moved left and right by dragging them, so it isn't so bad to add extra columns. In Quartus one has dozens for some of these reports and it is easy to drag the relevant columns leftmost and then also easier to sort on that column. Currently it isn't possible to sort on the columns using "/", even if the GUI supported it.
